datasets/_global/ohchr/ohchr_settlement.yml

```
@@ -0,0 +1,57 @@
+title: OHCHR Companies linked to illegal settlement in the West Bank
```

```suggestion
title: OHCHR List of companies linked to illegal settlement in the West Bank
```

Makes it a bit long, but it almost reads like these are _OHCHR Companies_

---

great documentation otherwise. I think we need an info URL and publisher details too.

---

also

```yaml
coverage:
  frequency: never
  end: 2023-12-18
```

as opposed to the usual `frequency: daily`